src/main/resources/application.properties

```diff
@@ -11,7 +11,7 @@ spring.jpa.open-in-view=false
 spring.datasource.hikari.connectionTimeout=45000
 spring.datasource.hikari.idleTimeout=600000
 spring.datasource.hikari.maxLifetime=30000
-spring.datasource.hikari.maximumPoolSize=2
+spring.datasource.hikari.maximumPoolSize=3
```

Comment:
Our database only allows max 10 connections, so I figured a max pool size of 3 allows the live site to be on all the time and 2 others concurrently running locally all while remaining under the 10 limit (and leaves 1 remaining just in case someone else needs to test something really quick, they might be able to locally set pool size to 1).

Reply:
Got it, makes sense - thanks!